README.md

```diff
@@ -343,6 +343,11 @@ const customCallbackManager = new CallbackManager([yourCustomCallback]);
 aiConfigRuntimeInstance.setCallbackManager(customCallbackManager);
 ```
 
+```python
+custom_callback_manager = CallbackManager([yourCustomCallback])
```

Comment:
What is yourCustomCallback? This code sample is incomplete. It shouldn't be pseudo code, fully runnable example

Reply:
updated to define callback and use it